/**:
  ros__parameters:
    # gyro_bias_threshold: 0.003 # [rad/s]
    # timer_callback_interval_sec: 0.1 # [sec]
    # diagnostics_updater_interval_sec: 0.5 # [sec]
    # straight_motion_ang_vel_upper_limit: 0.015 # [rad/s]
    timestamp_threshold: 0.1 # [s]
    data_num_threshold: 400 # [num]
    bias_change_threshold: 0.0050 # [rad/s]
    stddev_threshold: 0.00121 # [rad/s]
